### PR TITLE
Edit talks v2: fetch speakers

### PR DESCRIPTION
**Pull request by heiglandreas**
This PR allows to fetch already available speakers for a talk. It introduces 2 API-Endpoints:

* /v2.1/talks/[talk-id]/speakers will give a list of speakers for the talk [talk-id] with references to the user-url if the talk has been claimed
* /v2.1/speakers/[speaker-id] will give a list with one entry for the speaker [speaker-id].

Both results look like the following JSON where the list for the **/v2.1/talks/[talk-id]/speakers** can contain multiple entries and the list for the **/v2.1/speakers/[speaker-id]** will only contain 1 entry (the entry for the requested speaker-entry).

Depending on whether the user has claimed the talk the ```full_name``` and ```twitter_username``` entries will contain a value or not and if the talk is claimed the entry ```user_uri``` is available or not.

### CODE REVIEW

File: src/models/TalkSpeakerMapper.php [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    public function getDefaultFields() {
+        $fields = array(
+            "username" => "speaker_name",
+            "full_name" => "full_name",
+            "twitter_username" => "twitter_username"
+        );
+        return $fields;
+    }
+
+    public function getVerboseFields() {
```
Comment (chrishoult) [inline src/models/TalkSpeakerMapper.php]: Would it not be more future-proof to return `$this->getDefaultFields();`?

File: src/models/TalkSpeakerMapper.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+        // group by because of adding the user_attend join
+        $sql .= ' group by speaker.ID ';
+
+        // order by
+        if ($order) {
+            $sql .= ' order by ' . $order;
+        }
+
+        error_log($sql);
```
Comment (chrishoult) [inline src/models/TalkSpeakerMapper.php]: Is this left in intentionally? If so, you're missing more SQL below :)

File: src/models/TalkSpeakerMapper.php [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+        $countStmt = $this->_db->prepare($sql);
```
Comment (chrishoult) [inline src/models/TalkSpeakerMapper.php]: This isn't used...

**Comment by akrabat** [comment]
I'm closing this as the talk claims issue [JOINDIN-637](https://joindin.jira.com/browse/JOINDIN-637) overlaps.

**Pull request closed without merging**